app/controllers/support_interface/change_course_controller.rb

```diff
@@ -0,0 +1,44 @@
+module SupportInterface
+  class ChangeCourseController < SupportInterfaceController
+    def options
+      @pick_option_form = SupportInterface::ChangeCourseForm.new(application_form: application_form)
+    end
+
+    def pick_option
+      change_course_form = SupportInterface::ChangeCourseForm.new(params.require(:support_interface_change_course_form).permit(:change_type))
+
+      case change_course_form.change_type
+      when 'add_course'
+        redirect_to support_interface_add_course_to_application_path(application_form)
+      else
+        flash[:info] = "Sorry - we haven't built this feature yet"
+        redirect_to support_interface_change_course_path(application_form)
+      end
+    end
+
+    def select_course_to_add
```

Comment:
I think this controller will get a bit big when all 4 options are implemented so perhaps executing the individual options would be easier to manage as separate controllers? Can be looked at later I guess.

Reply:
Good point. I think the individual options might become quite big indeed, so I'll keep an eye on that.